youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,438 @@
+"PO-Revision-Date: 2015-07-24 04:37+EEST\n"
+"Last-Translator: Ali KOÇ <alikoc@protonmail.ch>\n"
+"Language-Team: tr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+
+#: mainframe.py:82
+msgid "URLs"
+msgstr "Url(ler)"
+
+#: mainframe.py:83
+msgid "Download"
+msgstr "İndirme"
+
+#: mainframe.py:84
+msgid "Update"
+msgstr "Güncelleme"
+
+#: mainframe.py:85 optionsframe.py:40
+msgid "Options"
+msgstr "Ayarlar"
+
+#: mainframe.py:86
+msgid "Error"
+msgstr "Hata"
+
+#: mainframe.py:87
+msgid "Stop"
+msgstr "Dur"
+
+#: mainframe.py:88
+msgid "Info"
+msgstr "Bilgi"
+
+#: mainframe.py:89
+msgid "Welcome"
+msgstr "Hoşgeldiniz"
+
+#: mainframe.py:90
+msgid "Successfully downloaded {0} url(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "{0} url(ler) {1} gün {2} saat {3} dakika {4} saniye içinde başarı ile indirildi"
+
+#: mainframe.py:92
+msgid "Downloads completed"
+msgstr "İndirmeler tamamlandı"
+
+#: mainframe.py:93
+msgid "Downloading {0} url(s)"
+msgstr "İnidiriliyor {0} url(ler)"
+
+#: mainframe.py:94
+msgid "Stopping downloads"
+msgstr "İndirmeler durduruluyor"
+
+#: mainframe.py:95
+msgid "Downloads stopped"
+msgstr "İndirmeler durduruldu"
+
+#: mainframe.py:96
+msgid "You need to provide at least one url"
+msgstr "En az bir url belirtmeniz gerek"
+
+#: mainframe.py:97
+msgid "Downloads started"
+msgstr "İndirmeler başlatıldı"
+
+#: mainframe.py:99
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "youtube-dl'nin en son sürümü indiriliyor. Lütfen bekleyin..."
+
+#: mainframe.py:100
+msgid "Youtube-dl download failed [{0}]"
+msgstr "Youtube-dl indirme başarısız"
+
+#: mainframe.py:101
+msgid "Youtube-dl downloaded correctly"
+msgstr "Youtube-dl başarılı olarak indirildi"
+
+#: mainframe.py:103
+msgid "Unable to open directory: '{dir}'. The specified path does not exist"
+msgstr "'{dir}' klasörüne erişilemiyor. Belirtilen yol geçerli değil"
+
+#: mainframe.py:105
+msgid "Error while shutting down. Make sure you typed the correct password"
+msgstr "Kapanırken hata oluştu. Doğru şifreyi girdinizden emin olun"
+
+#: mainframe.py:107
+msgid "Shutting down system"
+msgstr "Sistem kapatılıyor"
+
+#: mainframe.py:109
+msgid "Title"
+msgstr "Başlık"
+
+#: mainframe.py:110
+msgid "Size"
+msgstr "Boyut"
+
+#: mainframe.py:111
+msgid "Percent"
+msgstr "Yüzde"
+
+#: mainframe.py:112
+msgid "ETA"
+msgstr "TKZ"
+
+#: mainframe.py:113
+msgid "Speed"
+msgstr "Hız"
+
+#: mainframe.py:114
+msgid "Status"
+msgstr "Durum"
+
+#: optionsframe.py:42
+msgid "General"
+msgstr "Genel"
+
+#: optionsframe.py:43
+msgid "Video"
+msgstr "Video"
+
+#: optionsframe.py:44
+msgid "Audio"
+msgstr "Ses"
+
+#: optionsframe.py:45
+msgid "Playlist"
+msgstr "Çalma listesi"
+
+#: optionsframe.py:46
+msgid "Output"
+msgstr "Çıkış"
+
+#: optionsframe.py:47
+msgid "Subtitles"
+msgstr ""
+
+#: optionsframe.py:48
+msgid "Filesystem"
+msgstr "Alt yazı"
```

Comment:
Doesn't *altyazi* means subtitles? I believe this should be the translation for line 146